### DISCUSSION TITLE
Installing Docker and Anaconda Questions

### DISCUSSION THREAD

**Discussion by iismailiismail** [opensafely/documentation#319] [Installing OpenSAFELY] [answered]
I had some tech questions I need to know to install the above:

1) Docker:

For Open Safely do we need the Docker Desktop for Windows ? Does this also require WSL2 to be installed on the Windows device?

2) Anaconda:

Does installing Anaconda require admin rights to do this on Windows?

Would be very grateful if anyone can give some advice:)

**Comment by bloodearnest** [comment, marked as answer]
Hello! :)

Have you seen the installation guide? It has answers to these questions, and instructions for installation: 

https://docs.opensafely.org/getting-started/#3-setup-the-required-software

Note: the default instructions are for using the cloud based gitpod.io service, if you want to install the tools locally, click on the 'Windows (local)` tab.
